### ISSUE TITLE
This plugin needs to be updated to comply with Moodle privacy requirements.

### ISSUE THREAD

**Issue by ctam** [ucsf-ckm/moodle-filter_ucsfezproxy#14]
In Moodle 4.3, this plugin causes a core unit test error as follows:
```
core_privacy\privacy\provider_test::test_all_providers_compliant with data set "filter_ucsfezproxy" ('filter_ucsfezproxy', 'filter_ucsfezproxy\privacy\provider')
Failed asserting that false is true.

/home/runner/work/moodle/moodle/privacy/tests/privacy/provider_test.php:179
/home/runner/work/moodle/moodle/lib/phpunit/classes/advanced_testcase.php:81
```

**Comment by stopfstedt** [comment]
this has been fixed in the `MOODLE_43_DEVEL` branch already.

**Comment by stopfstedt** [comment]
fixed via #15
